client/cmd/dexc-desktop/snap/local/snapcraft.yaml.in

```diff
@@ -0,0 +1,102 @@
+name: dexc
+adopt-info: metainfo
+version: '$VERSION'
+license: BlueOak-1.0.0
+
+base: core22 
+grade: stable
+confinement: strict
+
+architectures:
+  - build-on: amd64
+compression: lzo
+
+plugs:
+  dbus-svc:
+    bus: system
+    interface: dbus
+    name: org.freedesktop.portal
+  dbus-statusnotifier:
+    bus: session
+    interface: dbus
+    name: org.kde.StatusNotifierWatcher
+  dbus-dbusmenu:
+    bus: session
+    interface: dbus
+    name: com.canonical.dbusmenu
+
+apps:
+  dexc:
+    common-id: org.decred.dcrdex
+    command: usr/bin/dexc
+    desktop: usr/share/applications/dexc.desktop
+    extensions: [gnome]
+    environment:
+      GTK_USE_PORTAL: "1"
+      #TMPDIR: $XDG_RUNTIME_DIR
+      #XDG_CURRENT_DESKTOP: Unity
+      #DISABLE_WAYLAND: 1
+    plugs: 
+      - home
+      - opengl
+      - x11
+      - desktop
+      - desktop-legacy
+      - network
+      - network-status
+      - browser-support
+      - screen-inhibit-control
+      - dbus-svc
+      - dbus-dbusmenu
+      - dbus-statusnotifier
```

Comment:
How is this list generated and what does it mean?

Reply:
This list defines the snap interfaces the app needs access to, basically they translate into AppArmor permissions.  It's assembled from the documentation and trial-and-error.

https://snapcraft.io/docs/interface-management